src/components/standalone-link/types.ts

```diff
@@ -0,0 +1,13 @@
+import { ReactElement } from 'react'
+import { TextProps } from 'components/text'
+
+export interface StandaloneLinkProps {
+  className?: string
+  color?: 'primary' | 'secondary'
+  href: string
+  icon: ReactElement
```

Comment:
nit: It could be nice to have a doc comment above this property with an example 💭

Reply:
(Also would be great to have the descriptions from the table in swingset in doc comments above the properties!)

Reply:
I can make both of these changes! 😊

Reply:
Added in https://github.com/hashicorp/dev-portal/pull/100/commits/2e8335efb90580638800593cdffd692b789624fe!